tests/integration_tests/performance/test_process_startup_time.py

```diff
@@ -75,6 +75,11 @@ def _test_startup_time(microvm):
     )
 
     assert cpu_startup_time_us > 0
+    # No specific reason to compare against 150ms or 10ms
+    # its just to catch issues like the ones introduced in PR
+    # https://github.com/firecracker-microvm/firecracker/pull/4305
+    assert startup_time_us < 150000
+    assert cpu_startup_time_us < 10000
```

Comment:
Instead of hardcoding values, can we do something like `initial = time.time() * 1000 * 1000` at the start of this function, then `end = time.time() * 1000 * 1000` after the `microvm.get_all_metrics()` call, and then 
 ```suggestion
    # No specific reason to compare against 150ms or 10ms
    # its just to catch issues like the ones introduced in PR
    # https://github.com/firecracker-microvm/firecracker/pull/4305
    assert startup_time_us < end - begin
    assert cpu_startup_time_us < end - begin
```
instead? This test the same regression (e.g. start time is not longer than the process has been alive in the first place), but without any magic numbers that might later on accidentally become SLAs